src/Sylius/Bundle/AdminBundle/Controller/Dashboard/StatisticsController.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Controller\Dashboard;
+
+use Sylius\Bundle\MoneyBundle\Formatter\MoneyFormatterInterface;
+use Sylius\Component\Core\Dashboard\DashboardStatisticsProviderInterface;
+use Sylius\Component\Core\Dashboard\Interval;
+use Sylius\Component\Core\Dashboard\SalesDataProviderInterface;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+
+final class StatisticsController
+{
+    /** @var EngineInterface */
+    private $templatingEngine;
+
+    /** @var DashboardStatisticsProviderInterface */
+    private $statisticsProvider;
+
+    /** @var SalesDataProviderInterface */
+    private $salesDataProvider;
+
+    /** @var MoneyFormatterInterface */
+    private $moneyFormatter;
+
+    public function __construct(
+        EngineInterface $templatingEngine,
+        DashboardStatisticsProviderInterface $statisticsProvider,
+        SalesDataProviderInterface $salesDataProvider,
+        MoneyFormatterInterface $moneyFormatter
+    ) {
+        $this->templatingEngine = $templatingEngine;
+        $this->statisticsProvider = $statisticsProvider;
+        $this->salesDataProvider = $salesDataProvider;
+        $this->moneyFormatter = $moneyFormatter;
+    }
+
+    public function __invoke(ChannelInterface $channel): Response
```

Comment:
The rule of thumb for me would be, that if we have more than one function in the class, none of them should be called `__invoke`. 
```suggestion
    public function renderStatistics(ChannelInterface $channel): Response
```